roles/windows/kubernetes/tasks/download_kubernetes_scripts.yml

```diff
@@ -0,0 +1,29 @@
+- name: Flannel | Download kubernetes hns.psm1
+  win_get_url:
+    url: "{{ download_links.download_kubernetes_hns }}"
+    dest: "{{ download_dest.hns }}"
+
+- name: Flannel | Download InstallImages.ps1
+  win_get_url:
+    url: "{{ download_links.download_kubernetes_install_images }}"
+    dest: "{{ download_dest.install_images }}"
+
+- name: Flannel | Download Dockerfile
+  win_get_url:
+    url: "{{ download_links.download_kubernetes_dockerfile }}"
+    dest: "{{ download_dest.dockerfile }}"
+
+- name: Flannel | Download stop.ps1
```

Comment:
we don't use powershell to manage kubelet, flannel nor kube-proxy, so we do not need these start/stop ps1